### PR TITLE
add wormhole UST and LUNA

### PR DESCRIPTION
**Pull request by YunSuk-Yeo**
Pull Request to add Terra native assets to MetaMask relayed via Wormhole.

Relevant Links
Terra
Official Website: https://terra.money/

Github: https://github.com/terra-money/core
Medium: https://medium.com/terra-money
Twitter: https://twitter.com/terra_money
Discord: https://discord.gg/bYfyhUT
Telegram: https://t.me/TerraLunaChat

Wormhole
Official Website: https://wormholenetwork.com
Github: https://github.com/certusone/wormhole
Medium: https://wormholecrypto.medium.com
Twitter: https://twitter.com/wormholecrypto
Discord: https://discord.gg/wormholecrypto
Telegram: https://t.me/wormholecrypto

### CODE REVIEW

**Review by MRabenda — APPROVED** [review]

**Comment by YunSuk-Yeo** [comment]
@MRabenda Sorry for late response, I fixed overlapping symbol problem.

**Comment by YunSuk-Yeo** [comment]
solved all test fail problems

**Comment by YunSuk-Yeo** [comment]
@MRabenda sorry to ping you again, but there was some updates.

**Comment by YunSuk-Yeo** [comment]
sorry, there should be more update. so please wait

**Comment by YunSuk-Yeo** [comment]
@MRabenda updates are completed

File: out [hunk 1]
```diff
@@ -0,0 +1,6143 @@
+
```
Comment (MRabenda) [inline out:1]: Please remove this file

Comment (YunSuk-Yeo) [reply out]: @MRabenda Done

**Pull request merged**